templates/web-application/src/RootRoutingErrorBoundary.tsx

```diff
@@ -10,14 +10,26 @@ function getErrorMessage(error: unknown) {
         : JSON.stringify(error);
 }
 
-export function RootErrorBoundary() {
+export function RootRoutingErrorBoundary() {
```

Comment:
I would prefer to keep the name `RootErrorBoundary` here.

As for the component extracted for the story, I would name it "UnmanagedError".